anomalib/models/rkde/config.yaml

```diff
@@ -0,0 +1,117 @@
+dataset:
+  name: ucsd #options: [mvtec, btech, folder]
+  format: ucsdped
+  path: ./datasets/ucsd
+  category: UCSDped2
+  task: detection
+  clip_length_in_frames: 1
+  frames_between_clips: 10
+  image_size: [240, 360] # RKDE region- and feature extractor apply custom resizing
```

Comment:
Yeah, both the region extractor and the feature extractor both apply their own set of transforms on the input images. So it's best to keep the input size the same here as the original images on the file system to prevent losing any information.